code/HabitTracker/app/src/main/java/com/example/habittracker/activities/profile/ProfileActivity.java

```diff
@@ -70,7 +73,37 @@ public void onClick(View view) {
         logout_button.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                Intent intent = new Intent(getApplicationContext(), ProfileLoginActivity.class);
+                // Log the user out
+
+                // Set the global user instance
+                SharedInfo.getInstance().clearCurrentUser();
+
+                // Switch to login screen
+                Intent intent = new Intent(getApplicationContext(), LoginActivity.class);
+                startActivity(intent);
+            }
+        });
+
+        Button delete_button = findViewById(R.id.delete_account);
```

Comment:
Jie, can you comment out the delete part for now?

Reply:
Even if you make the button invisible, actions will still get triggered if the user unknowingly presses the invisible button. I was testing out your program and this caused the app to crash.